sycl/test-e2e/Adapters/level_zero/uuid.cpp

```diff
@@ -25,6 +25,15 @@ int main() {
   auto zedev = sycl::get_native<sycl::backend::ext_oneapi_level_zero>(dev);
   ze_device_properties_t device_properties{};
   device_properties.stype = ZE_STRUCTURE_TYPE_DEVICE_PROPERTIES;
+
+  // Initialize Level Zero driver is required if this test is linked
+  // statically with Level Zero loader, the driver will not be init otherwise.
+  ze_result_t result = zeInit(ZE_INIT_FLAG_GPU_ONLY);
+  if (result != ZE_RESULT_SUCCESS) {
+    std::cout << "zeInit failed\n";
+    return 1;

```

Comment:
\+ some small enhancement here and below:
```suggestion
    return result;
```

Reply:
Maybe print this `result` value to output for the future easier debugging session, or return it in other way?